### ISSUE TITLE
Teltonika KLINE data problem

### ISSUE THREAD

**Issue by mskultety** [traccar/traccar#3827]
Hi, 

We have problem with getting data from tachograph. 

We are using Teltonika FM6320 connected to DAF can fms with tachograph. 

From traccar server log i can see this packet: 

```
00000000000000A708010000016269E7D9A8000A5A0F0A1CBF8F3300880046120000001C0801014F005100550F740073007801790103430000440000426F980B540000000056000045275700000047580000022659000000005D0000000068000003D07100007355870000000288000000008A000045270669584C5241534834336A30304731363538326B3600FFFFFF0000008155412055414430308230303039383236368330303000000000000100008396
```

When i decode it with teltonika decoder i've got theese values: 

![image](https://user-images.githubusercontent.com/37867866/38031583-d7584e7e-329b-11e8-9628-efc657c3a33d.png)

i am interested in io129, io130 and io131. Theese values are fms tachograph driver card.

but in traccar traccar i have values: 
io129: 6143226917249364000
io130: 3472328335016736300
io131: 3472328089260785700

also io105-107 are decoded wrong. It seems it has problem with decoding 8byte ios

**Comment by floman321** [comment]
+1

**Comment by tananaev** [comment]
I have tested latest code and everything seems to be decoded correctly:
```
25 = {LinkedHashMap$Entry@3009} "io105" -> "6362550814091260979"
26 = {LinkedHashMap$Entry@3010} "io106" -> "3472353589391079474"
27 = {LinkedHashMap$Entry@3011} "io107" -> "3891391553008041984"
28 = {LinkedHashMap$Entry@3012} "io129" -> "6143226917249364016"
29 = {LinkedHashMap$Entry@3013} "io130" -> "3472328335016736310"
30 = {LinkedHashMap$Entry@3014} "io131" -> "3472328089260785664"
```

**Comment by mskultety** [comment]
yesterday we get data from traccar trough API and it is working correctly. It is problem with traccar web. Data is stored in traccar correctly but is shown in web incorrectly.

**Comment by tananaev** [comment]
That's a limitation of JavaScript. It can't represent full 64-bit integer values. There is nothing we can do about it.

**Comment by tananaev** [comment]
Relevant explanation:

https://stackoverflow.com/a/17320771/2548565

**Comment by tananaev** [comment]
Fixed in 4a5eece13341722fbd20d299ce28bbbcdfc146bd.

**Comment by karls4** [comment]
@mskultety Could you provide me with some details on your setup?

I helping someone setup a system at the minute in lorries and have the tacograph info would be great.

My email is in my profile if you could send me a message that would be great. Thanks
